queue/common.go

```diff
@@ -68,6 +68,26 @@ func NewBroker(uri string) (Broker, error) {
 // TxCallback is a function to be called in a transaction.
 type TxCallback func(q Queue) error
 
+// RepublishConditionFunc is a function used to filter jobs to republish.
+type RepublishConditionFunc func(job *Job) bool
+
+type republishConditions []RepublishConditionFunc
+
+func (c republishConditions) comply(job *Job) bool {
```

Comment:
Couldn't this be simplified?

```go
if len(c) == 0 {
  return true
}

for _, condition := range c {
  if condition(job) {
    return true
  }
}

return false
```

Otherwise LGTM

Reply:
yes, it looks like much better :+1: